examples/components/NER/upload_data_to_silos/spec.yaml

```diff
@@ -0,0 +1,40 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+
+outputs:
+  silo0_train_data:
+    type: uri_folder
+    mode: mount
+    path: azureml://datastores/datastore_silo0_westus/paths/multinerd/train/
+  silo0_test_data:
+    type: uri_folder
+    mode: mount
+    path: azureml://datastores/datastore_silo0_westus/paths/multinerd/test/
+
+  silo1_train_data:
+    type: uri_folder
+    mode: mount
+    path: azureml://datastores/datastore_silo1_francecentral/paths/multinerd/train/
+  silo1_test_data:
+    type: uri_folder
+    mode: mount
+    path: azureml://datastores/datastore_silo1_francecentral/paths/multinerd/test/
+
+  silo2_train_data:
+    type: uri_folder
+    mode: mount
+    path: azureml://datastores/datastore_silo2_brazilsouth/paths/multinerd/train/
```

Comment:
Does this actually work? As per my understanding according to discussion with @jfomhover if an orchestrator is able to write to silo storage this is a bug.

Reply:
This works for the open setup but we still need to figure out how to do this for a secure setup. IMO, this situation would never come in a real scenario/production where we need to bifurcate data and then upload it to multiple silos. If we already have data in one place, then that's no more an FL problem. The other scenario would be if the data is already partitioned then we can provide a script later that would run in silo computes and upload data to their respective datastores.

Reply:
This is interesting - how is it possible that it works with following UID on orchestrator?
![image](https://user-images.githubusercontent.com/9350520/203395336-6a8cff56-2005-437c-9b07-05e7c04c6d02.png)
It should not have access to other blobstorages. Do you know why is that happening?

Reply:
This might be happening because I manually created GPU computes and all the data stores are under one storage account. Happy to elaborate on this on Teams if needed!

Reply:
The problem with such setup though is that storage account has assigned location and thus all the data will be hosted in the same region, right? Unless I may be missing something?

Teams is also a way but as this is something we all need to solve for industrial examples, it might be better to have it in a shared space.

Reply:
Follow up: Discussed in a call with a conclusion that we will try to create a compute that can access all the blob storages and the data will be uploaded to corresponding regions.

Reply:
Created a compute that has access to the silos' storage accounts and updated the documentation accordingly.